security-method/src/aotTest/java/com/example/security/method/SecurityMethodApplicationAotTests.java

```diff
@@ -19,22 +19,45 @@ void anonymousCanCallOnlyAnonymousMethod(AssertableOutput output) {
 			assertThat(output)
 					.hasSingleLineContaining("testAnonymous(): protectedService.anonymous() worked as anonymous")
 					.hasSingleLineContaining("testUser(): protectedService.user() failed as anonymous")
-					.hasSingleLineContaining("testAdmin(): protectedService.admin() failed as anonymous");
+					.hasSingleLineContaining("testAdmin(): protectedService.admin() failed as anonymous")
```

Comment:
I would prefer it if `@Secured` and JSR250 are separate tests (if these features are separate code paths in Spring Security). This way we can look at the failing tests to see which features are broken.